packages/fxa-content-server/app/tests/spec/lib/survey-filter.js

```diff
@@ -577,33 +577,6 @@ describe('lib/survey-filter', () => {
     });
   });
 
-  describe('userAgentChecks', () => {
```

Comment:
Assuming this was just part of getting started on the tests, but why was this one removed?